cpp/src/arrow/acero/asof_join_node_test.cc

```diff
@@ -1380,7 +1380,7 @@ T GetEnvValue(const std::string& var, T default_value) {
 }  // namespace
 
 TEST(AsofJoinTest, BackpressureWithBatchesGen) {
-  int num_batches = GetEnvValue("ARROW_BACKPRESSURE_DEMO_NUM_BATCHES", 10);
+  int num_batches = GetEnvValue("ARROW_BACKPRESSURE_DEMO_NUM_BATCHES", 2);
```

Comment:
Not sure we want to modify this parameter? cc @westonpace

Reply:
oops, sorry left in from testing.  Reverting.